policy/README.md

```diff
@@ -0,0 +1,354 @@
+
+Language tag values will be per [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt).
+
+Example for a decreased speed-limit rule for Venice Beach on weekends:
+
+```
+	"messages": {
+	            "en-US": "Remember to stay under 10 MPH on Venice Beach on weekends!”,
+	            "es-US": "¡Recuerda mantener por debajo 10 millas por hora en Venice Beach los fines de semana!"
+	        },
+```
+
+<a name="value-url"></a>
+### Value URL
+
+An Agency may wish to provide dynamic or global rules, e.g. "Within 300 yards of the stadium, 1000 total extra scooters may be deployed, across all Provider(s)."  In this case, compliance is not computable from the information available to a single Provider.  The Agency would provide a specified endpoint to get the current count of vehicles in the service-area, so that individual Providers could decide whether adding some number to those present can be done.  
+
+There are potential complexities that are as-yet unaddressed, such as the latency between a Provider deciding to enter such an area, and the number of vehicles in the area when the scooters arrive.  As this specification is Alpha, feedback and refinements are welcome.
+
+The payload returned from a GET call to the `value_url` will have the following immutable fields:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `value`      | integer   | R   | Value of whatever the rule measures |
+| `timestamp`  | timestamp | R   | Timestamp the value was recorded |
+| `policy_id`  | UUID      | R   | Relevant `policy_id` for reference |
+
+MDS will likely include some sort of streaming mechanism in the upcoming releases that may supplant or replace this mechanism.
+
+<a name="endpoints"></a>
+## Endpoints
+
+The provider-facing Policy API consists of the following two endpoints.
+
+`GET /policies`
+
+Parameters:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `start_time` | timestamp | O   | Earliest effective date, default=effective now |
+| `end_time`   | timestamp | O   | Latest effective date, default=all future |
+
+Note: provider_id is an implicit parameter and will be encoded in the authentication key
+
+Returns: List of policy objects effective during the timespan described by greater than or equal to `start_time` and less than or equal to `end_time`
+
+Policies will be returned in order of effective date (see Policy Schema, below), with pagination as in Agency or Provider.
+
+`GET /policies/{id}`
+
+Returns: One policy object whose id is `id`
+
+Parameters: none
+
+`GET /geographies/{id}`
+
+Returns: One immutable GeoJSON FeatureCollection with an embedded UUID identifier property of `id`
+
+Parameters: none
+
+Note: Intentionally omitted `GET /geographies` until a compelling use-case can be identified.
```

Comment:
From a front-end perspective it would be useful to be able to retrieve a number of geographies in a single request.  This would facilitate, e.g. showing multiple the geographies in a map view quickly.